scripts/migrations/2020-08-update_migration_created_date.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+import json
+import uuid
+from botocore.exceptions import ClientError
+from deepdiff import DeepDiff
+
+from common import get_aws_resource, scan_table
+
+READ_ONLY_ROLE_ARN = "arn:aws:iam::975596993436:role/storage-read_only"
+dynamodb = get_aws_resource("dynamodb", role_arn=READ_ONLY_ROLE_ARN)
+s3 = get_aws_resource("s3", role_arn=READ_ONLY_ROLE_ARN)
+
+vhs_table = "vhs-storage-staging-manifests-2020-07-24"
+
+backfill_vhs_table = "vhs-storage-staging-manifests-2020-08-19"
+
+errors = {}
+
+
+def record_error(id, version, err):
+    print(f"\033[91mError while updating {id}/{version}: {err}")
+    errors[f'{id}/{version}'] = err
+
+def get_bucket_key(item):
+    try:
+        bucket = item["payload"]["namespace"]
+        key = item["payload"]["path"]
+    except KeyError:
+        try:
+            bucket = item["payload"]["bucket"]
+            key = item["payload"]["key"]
+        except:
+            record_error(item["id"], item["version"], f"Cannot find s3 bucket, key in {item}")
+        else:
+            return bucket, key
+    else:
+        return bucket, key
+
+
+def get_vhs_json(id, version, bucket, key):
+    try:
+        return json.loads(s3.Object(bucket, key).get()["Body"].read().decode("utf-8"))
+    except ClientError as e:
+        record_error(id, version, f"Cannot read s3 entry for {id}: {item}: {e}")
+
+
+def put_vhs_json(id, version, bucket, content):
+    bb = uuid.uuid4()
+    key = f"{id}/{version}/{bb}"
```

Comment:
```suggestion
    key = f"{id}/{version}/{bb}.json"
```

We now stick `.json` on the end of these keys, so it's slightly easier to deal with if you're browsing the VHS by hand.

Reply:
ah thanks, missed that